resources/js/Pages/Answer/Index.vue

```diff
@@ -0,0 +1,15 @@
+<script setup lang="ts">
+import {type Answer} from '@/Types/Answer'
+import Show from './Show.vue'
+
+const props = defineProps<{
+  answers: Answer[]
+}>()
+</script>
+
+<template>
+  <h1 v-if="props.answers.length > 0" class="text-xl">ANSWERS</h1>
+  <div v-for="item in props.answers" :key="item.id">
+    <Show :answer="item" />
+  </div>
+</template>
```

Comment:
In my opinion, the frontend should be made in a separate pull request, as it will be time-consuming. This pull request is currently blocking further progress on the tests feature (#9). 

Should I remove the frontend files and the `index` and `show` methods from the controllers, or can I leave them in this pull request?

Reply:
u can leave that here, its only my question about frontend.